### PR TITLE
add a license, minor clean up of README

### PR DESCRIPTION
**Pull request by GGAlanSmithee**
This PR adds a license. I also took the opportunity to do a minor cleanup of the README file. More changes to come in that area though.

### CODE REVIEW

**Review by brokenprogrammer — APPROVED** [review]
LGMT! Ser ut som det vi snackade om, snyggt. 👍

File: DEVELOPER.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+## Send translated messages from backend
+
+All errors thrown by the backend should already be translated so these can be shown directly as snackbars or be embeded in the UI!
+
+**do**
+
+```CSharp
+    return  BadRequest(App.Error_BadRequest);
```
Comment (mrmaglet) [inline DEVELOPER.md:110]: 🎉

**Review by mrmaglet — APPROVED** [review]
Looks good to me. In regards to the actual licens - I trust this has been cleared/approved already. 👍

**Comment by GGAlanSmithee** [comment]
Thanks for the review guys, I will close this for now, we can re-open later.

**Pull request closed without merging**